content/blog/2023/08/28/2023-08-28-renovate-bote-probe-blog.adoc

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: Renovate Probe
+tags:
+- gsoc
+- gsoc2023
+- healthscore
+- probes
+- plugin
+authors:
+- jagruti
+opengraph:
+  image: /images/gsoc/2023/jagruti/renovate-probe.jpg
+sig:
+- gsoc
+---
+
+== Introduction
+
+Bots are used to automate tasks in a plugin.
+The Plugin Health Scoring System tracks usage of two bots - link:https://github.com/dependabot[Dependabot] and link:https://docs.renovatebot.com/[Renovate].
+
+Dependabot looks for dependencies that are outdated in a plugin. It can also raise a security alert if a security breach is found like when the code depends on an insecure package.
+
+Renovate provides automated dependency updates. It can also schedule PRs, and customize behavior using configuration files.
+
+
+== Importance of the probe
+
+This a community requested probe to track the usage of Renovate. Bot tracking probes help in identifying which bot is used most by the plugins. In the future, the probe will aid in decision-making when the need arises to add or remove a bot.
+
+The pull requests created by Renovate will be counted when scoring the probe.
+
+== Challenges
+
+The renovate probe required creating an `abstract` class and reconfiguring the Dependabot class to use the new `abstract` class.
+
+`Dependabot` and `Renovate` probes both had similar functionality: finding their respective configuration in `GitHub` workflow.
```

Comment:
```suggestion
`Dependabot` and `Renovate` probes both had similar functionality: finding their respective configuration in `GitHub` workflow.
```
This reads as though the similar functionality is how to find their respective configurations. Is the similarity in how they work, where they are found, or something else entirely?

Reply:
The similarity is in "where they are found". The configuration files for both are found in the `GitHub` workflow. 

And if the configuration exists means the bot is used.

Reply:
Ah okay, thank you very much for clarifying @Jagrutiti !